### PR TITLE
Always installer-provisioned

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/19919

### CODE REVIEW

**Comment by jboxman** [comment]
@kalexand-rh, how's this? It reads kind of strange with _installer installation_.

File: modules/accessing-hosts-on-aws.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // * networking/accessing-hosts.adoc
 
 [id="accessing-hosts-on-aws_{context}"]
-= Accessing hosts on Amazon Web Services (AWS) after a full-stack automation (IPI) installation
+= Accessing hosts on Amazon Web Services (AWS) after a full-stack automation installer-provisioned installation
```
Comment (kalexand-rh) [inline modules/accessing-hosts-on-aws.adoc]: I might say "Accessing hosts on Amazon Web Services in an installer-provisioned infrastructure cluster"

I don't see AWS in the body of the module, so you don't need to define the initialism.

**Pull request merged**